packages/invoice-template/src/InvoiceBase/index.tsx

```diff
@@ -0,0 +1,165 @@
+import * as React from "react";
+import { ReactNode, FunctionComponent } from "react";
+import { format } from "date-fns";
+import styled from "styled-components";
+import LabeledField from "../common/LabeledField";
+import Table, { TableProps } from "../common/Table";
+
+const mainTableClassName = "main-table";
+const taxesTableClassName = "taxes-table";
+
+export type TableDataProp = Omit<TableProps, "className">;
+
+export type LabeledField<T> = {
+    label: string;
+    value: T;
+};
+
+export type LocalizatonOptions = {
+    dateFormat: string;
+    documentDate: LabeledField<Date>;
+    sellDate: LabeledField<Date>;
+    dueDate: LabeledField<Date>;
+    paymentMethod: LabeledField<ReactNode>;
+    seller: LabeledField<ReactNode>;
+    buyer: LabeledField<ReactNode>;
+    total: LabeledField<ReactNode>;
+};
+
+export type InvoiceBaseProps = {
+    logo: ReactNode;
+    name: ReactNode;
+    localizatonOptions: LocalizatonOptions;
+    mainTable: TableDataProp;
```

Comment:
Not sure if `mainTable` is self explainatory. Maybe `invoiceItems`? Or `invoiceItemsTable`?